model/src/guild/auto_moderation/mod.rs

```diff
@@ -0,0 +1,25 @@
+//! Auto Moderation is a feature which allows each guild to set up rules that
+//! trigger based on some criteria. For example, a rule can trigger whenever a
+//! message contains a specific keyword.
+//!
+//! Rules can be configured to automatically execute actions whenever they
+//! trigger. For example, if a user tries to send a message which contains a
+//! certain keyword, a rule can trigger and block the message before it is sent.
+
+#![deny(missing_docs)]
+
+mod action;
+mod action_metadata;
+mod action_type;
```

Comment:
These modules are tiny, how about merging them? (No need for Java style 1 public item / module)